README.md

```diff
@@ -167,6 +171,8 @@ the following:
 5. `OpenOCD` - Start the openOCD server
 6. `BuildFlash` - Execute a build followed by a flash command.
 
+Note that for OpenOCD tasks you need to define OPENOCD_SCRIPTS in your system environment variables with openocd scripts folder path.
```

Comment:
Shouldn't it get defined automatically from the onboarding process? It is listed in tools.json.

Reply:
Yes it is, but it is saved within idf.customExtraVars which is not directly accessible from tasks.json. Will cook some parsing function in the extension to make it work.

Reply:
PTAL at [PR 18](https://github.com/espressif/vscode-esp-idf-extension/pull/18/files#diff-45327f86d4438556066de133327f4ca2)